plugins/modules/grafana_folder.py

```diff
@@ -0,0 +1,252 @@
+    'supported_by': 'community',
+    'metadata_version': '1.1'
+}
+
+DOCUMENTATION = '''
+---
+module: grafana_folder
+author:
+  - Rémi REY (@rrey)
+version_added: "2.10"
+short_description: Manage Grafana Folders
+description:
+  - Create/update/delete Grafana Folders through the Folders API.
+  - Also allows to add members in the team (if members exists).
+  - The Folders API is only available starting Grafana 5 and the module will fail if the server version is lower than version 5.
+options:
+  url:
+    description:
+      - The Grafana URL.
+    required: true
+    type: str
+    aliases: [ grafana_url ]
+  name:
+    description:
+      - The title of the Grafana Folder.
+    required: true
+    type: str
+    aliases: [ title ]
+  url_username:
+    description:
+      - The Grafana user for API authentication.
+    default: admin
+    type: str
+    aliases: [ grafana_user ]
+  url_password:
+    description:
+      - The Grafana password for API authentication.
+    default: admin
+    type: str
+    aliases: [ grafana_password ]
+  grafana_api_key:
+    description:
+      - The Grafana API key.
+      - If set, C(url_username) and C(url_password) will be ignored.
+    type: str
+  state:
+    description:
+      - Delete the members not found in the C(members) parameters from the
+      - list of members found on the Folder.
+    default: present
+    type: str
+    choices: ["present", "absent"]
+  use_proxy:
+    description:
+      - If C(no), it will not use a proxy, even if one is defined in an environment variable on the target hosts.
+    type: bool
+    default: yes
+  client_cert:
+    description:
+      - PEM formatted certificate chain file to be used for SSL client authentication.
+      - This file can also include the key as well, and if the key is included, I(client_key) is not required
+    type: path
+  client_key:
+    description:
+      - PEM formatted file that contains your private key to be used for SSL client authentication.
+      - If I(client_cert) contains both the certificate and key, this option is not required.
+    type: path
+  validate_certs:
+    description:
+      - If C(no), SSL certificates will not be validated.
+      - This should only set to C(no) used on personally controlled sites using self-signed certificates.
+      - Prior to 1.9.2 the code defaulted to C(no).
+    type: bool
+    default: yes
+'''
+
+EXAMPLES = '''
+---
+- name: Create a folder
+  grafana_folder:
+      url: "https://grafana.example.com"
+      grafana_api_key: "{{ some_api_token_value }}"
+      title: "grafana_working_group"
+      state: present
+
+- name: Delete a folder
+  grafana_folder:
+      url: "https://grafana.example.com"
+      grafana_api_key: "{{ some_api_token_value }}"
+      title: "grafana_working_group"
+      state: absent
+'''
+
+RETURN = '''
+---
+folder:
+    description: Information about the Folder
+    returned: On success
+    type: complex
+    contains:
+      TODO
```

Comment:
This seems to be causing `sanity` to fail
ERROR: plugins/modules/grafana_folder.py:0:0: return-syntax-error: RETURN.folder.contains: not a valid value for dictionary value @ data['folder']['contains']. Got 'TODO'